api/person/config/routes.json

```diff
@@ -47,6 +47,14 @@
       "config": {
         "policies": []
       }
+    },
+    {
+      "method": "POST",
```

Comment:
basicamente eu to criando uma rota chamada login na raiz do seu server aqui. tipo: localhost:1337/login